docs/en/operations/server-configuration-parameters/settings.md

```diff
@@ -1553,6 +1585,81 @@ For more details, see [GraphiteMergeTree](../../engines/table-engines/mergetree-
 </graphite_rollup_example>
 ```
 
+## google_protos_path
+
+Defines a directory containing proto files for Protobuf types.
+
+Example:
+
+```xml
+<google_protos_path>/usr/share/clickhouse/protos/</google_protos_path>
+```
+
+## http_handlers
+
+Allows using custom HTTP handlers. 
+To add a new http handler simply add a new `<rule>`. 
+Rules are checked from top to bottom as defined,
+and the first match will run the handler.
+
+The following settings can be configured by sub-tags:
+
+| Sub-tags             | Definition                                                                                                                                        |
+|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+| `url`                | To match the request URL, you can use the 'regex:' prefix to use regex match (optional)                                                           |
+| `methods`            | To match request methods, you can use commas to separate multiple method matches (optional)                                                       |
+| `headers`            | To match request headers, match each child element (child element name is header name), you can use 'regex:' prefix to use regex match (optional) |
+| `handler`            | The request handler                                                                                                                               |
+| `empty_query_string` | Check that there is no query string in the URL                                                                                                    |
+
+`handler` contains the following settings, which can be configured by sub-tags:
+
+| Sub-tags           | Definition                                                                                                                                                            |
+|--------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `url`              | A location for redirect                                                                                                                                               |
+| `type`             | Supported types: static, dynamic_query_handler, predefined_query_handler, redirect                                                                                    | 
+| `status`           | Use with static type, response status code                                                                                                                            |
+| `query_param_name` | Use with dynamic_query_handler type, extracts and executes the value corresponding to the <query_param_name> value in HTTP request params                             |
```

Comment:
Looks like this is an issue, I think we probably need to enclose `<query_param_name>` in back, ticks or quotes. Noticed build failures for docs, for [example](https://s3.amazonaws.com/clickhouse-test-reports/PRs/76181/e60da042ed03066f95e14a99f1e54aab17c4aa1b//docs_check/docs_build/run.log). I'll try to fix in: #76194
```suggestion
| `query_param_name` | Use with dynamic_query_handler type, extracts and executes the value corresponding to the `<query_param_name>` value in HTTP request params                             |
```

Reply:
> Looks like this is an issue, I think we probably need to enclose `<query_param_name>` in back, ticks or quotes. Noticed build failures for docs, for [example](https://s3.amazonaws.com/clickhouse-test-reports/PRs/76181/e60da042ed03066f95e14a99f1e54aab17c4aa1b//docs_check/docs_build/run.log). I'll try to fix in: #76194

That was an issue and there were also some issues with tags which contained `{`, `}`. Have PRd a fix with https://github.com/ClickHouse/ClickHouse/pull/76134

Reply:
yeah, thank you. I didn't notice earlier.